models/model.js

```diff
@@ -69,4 +69,23 @@ function selectAllArticles  () {
          })
       }
 
-      module.exports = { insertCommentToArticle, selectArticleidComment, selectAllArticles,  selectArticleId,  selectAllTopics, selectAllUsers}
\ No newline at end of file
+      
+     function deleteComment(comment_id) {
+
+      if (isNaN(comment_id)) {
+         return Promise.reject({ status: 400, msg: 'Bad request' });
+       }
+     
```

Comment:
One thing I'd like you to consider is whether PSQL would throw a error of it's own for articleIds that aren't integers - it's worth removing your own error handling here and seeing what happens.
Where possible we want psql to do the work for us